src/diffusers/schedulers/scheduling_utils.py

```diff
@@ -32,9 +32,13 @@ class SchedulerOutput(BaseOutput):
         prev_sample (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)` for images):
             Computed sample (x_{t-1}) of previous timestep. `prev_sample` should be used as next model input in the
             denoising loop.
+        pred_original_sample (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)` for images):
+            The predicted denoised sample (x_{0}) based on the model output from the current timestep.
+            `pred_original_sample` can be used to preview progress or for guidance.
     """
 
     prev_sample: torch.FloatTensor
+    pred_original_sample: Optional[torch.FloatTensor] = None
```

Comment:
I think we should move it to the individual schedulers' outputs (i.e. create an output class for every scheduler that can return `pred_original_sample`, similar to `KarrasVeOutput`).

Otherwise if another output class inherits from `ShedulerOutput` then we get stuff like `TypeError: non-default argument 'state' follows default argument` because `pred_original_sample` defaults to `None` here.
https://github.com/huggingface/diffusers/actions/runs/3104648149/jobs/5029362292